src/Foundation/NSUrlSessionHandler.cs

```diff
@@ -148,7 +151,7 @@ public NSUrlSessionHandler () : this (CreateConfig ())
 		[CLSCompliant (false)]
 		public NSUrlSessionHandler (NSUrlSessionConfiguration configuration)
 		{
-			if (configuration == null)
+			if (configuration is null)
 				throw new ArgumentNullException (nameof (configuration));
```

Comment:
Sebastien recommended that we change the ArgumentNullException's as well in these Nullability PRs
https://github.com/xamarin/xamarin-macios/wiki/Nullability#how-1

Reply:
@spouliot legacy is alive and keeps going strong ❤️

Reply:
@tj-devel709 good point, but I'm going to do that in another PR, since I have a third PR waiting for this one.